now.redirect.json

```
@@ -0,0 +1,14 @@
+{
+    "name": "hospice-hope-contemplating-action-redirect",
+    "scope": "ca-hospice-hope",
+    "alias": ["hospicehopestories.org.uk"],
```

```suggestion
    "alias": ["hopestories.org.uk"],
```
The URL they now want is this